test/e2e/e2e_test.go

```diff
@@ -240,24 +210,145 @@ func TestLowNodeUtilization(t *testing.T) {
+	for i := 0; i < 5; i++ {
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("lnu-pod-%v", i),
+				Namespace: testNamespace.Name,
+				Labels:    map[string]string{"test": "node-utilization", "name": "test-rc-node-utilization"},
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{{
+					Name:            "pause",
+					ImagePullPolicy: "Never",
+					Image:           "kubernetes/pause",
+					Ports:           []v1.ContainerPort{{ContainerPort: 80}},
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceCPU: *resource.NewMilliQuantity(tenthOfCpu, resource.DecimalSI),
+						},
+						Requests: v1.ResourceList{
+							v1.ResourceCPU: *resource.NewMilliQuantity(tenthOfCpu, resource.DecimalSI),
+						},
+					},
+				}},
+				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								{
+									MatchFields: []v1.NodeSelectorRequirement{
+										{Key: "metadata.name", Operator: v1.NodeSelectorOpIn, Values: []string{workerNodes[0].Name}},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+
+		t.Logf("Creating pod %v in %v namespace for node %v", pod.Name, pod.Namespace, workerNodes[0].Name)
+		_, err := clientSet.CoreV1().Pods(pod.Namespace).Create(ctx, pod, metav1.CreateOptions{})
+		if err != nil {
+			t.Logf("Error creating LNU pods: %v", err)
+			if err = clientSet.CoreV1().Pods(pod.Namespace).DeleteCollection(ctx, metav1.DeleteOptions{}, metav1.ListOptions{
+				LabelSelector: labels.SelectorFromSet(labels.Set(map[string]string{"test": "node-utilization", "name": "test-rc-node-utilization"})).String(),
+			}); err != nil {
+				t.Fatalf("Unable to delete LNU pods: %v", err)
+			}
+			return
+		}
+	}
+
+	t.Log("Creating RC with 5 replicas owning the created pods")
+	rc := RcByNameContainer("test-rc-node-utilization", testNamespace.Name, int32(5), map[string]string{"test": "node-utilization"}, nil, "")
 	if _, err := clientSet.CoreV1().ReplicationControllers(rc.Namespace).Create(ctx, rc, metav1.CreateOptions{}); err != nil {
-		t.Errorf("Error creating deployment %v", err)
+		t.Fatalf("Error creating RC %v", err)
 	}
+	defer deleteRC(ctx, t, clientSet, rc)
+	waitForRCPodsRunning(ctx, t, clientSet, rc)
 
-	evictPods(ctx, t, clientSet, nodeInformer, nodes, rc)
-	deleteRC(ctx, t, clientSet, rc)
+	// Run LowNodeUtilization strategy
+	evictionPolicyGroupVersion, err := eutils.SupportEviction(clientSet)
+	if err != nil || len(evictionPolicyGroupVersion) == 0 {
+		t.Fatalf("%v", err)
+	}
+	podEvictor := evictions.NewPodEvictor(
+		clientSet,
+		evictionPolicyGroupVersion,
+		false,
+		0,
+		nodes,
+		true,
+		false,
+		false,
+	)
+
+	podsOnleastUtilizedNode, err := podutil.ListPodsOnANode(ctx, clientSet, workerNodes[0], podutil.WithFilter(podEvictor.Evictable().IsEvictable))
```

Comment:
isn't `workerNodes[0]` the most utilized node? That's the node you added those 5 pods on above

Reply:
Right, I forgot to rename `podsOnleastUtilizedNode` to `podsOnMostUtilizedNode`